openmdao/docs/features/model_visualization/n2_basics.rst

```diff
@@ -0,0 +1,116 @@
+.. _n2_basics:
+
+******************************************
+Basics of Creating N2 Model Visualizations
+******************************************
+
+An OpenMDAO model can have a number of connections, and several different residuals being converged.
+Trying to keep track of all the connections in your head can be a bit challenging, but OpenMDAO offers
+some visualization tools to help see what's going on. This page explains the basics of generating an N2 diagram
+either from the command line or from a script.
+
+An N2 diagram, also known as an N-squared diagram, is a diagram in the shape of a matrix, representing functional or
+physical interfaces between system elements. It is used to systematically identify, define, tabulate, design, and
+analyze functional and physical interfaces. It applies to system interfaces and hardware and/or software interfaces.
+For more information, see N2_chart_.
+
+.. _N2_chart: https://en.wikipedia.org/wiki/N2_chart
+
+For this page, we will be using this code example. Notice that there is an error in this code because one of the
+connection lines has been commented out. This was done to show how the N2 diagram can help point out unconnected
+inputs quickly.
+
+.. embed-code::
+    ../test_suite/scripts/circuit_with_unconnected_input.py
+
+
+From the Command Line
+---------------------
+
+.. _om-command-view_n2:
+
+Generating an N2 diagram for a model from the command line is easy. First, you need a Python script that contains
+and runs the model.
+
+.. note::
+
+    To get into more details, if :code:`final_setup` isn't called in the script (either directly or as a result of :code:`run_model`
```

Comment:
could also mention here, as with the command line xdsm viewer, the model will not actually run and the script will terminate after final_setup.

Reply:
Also done.